### ISSUE TITLE
wcag sidebar

### ISSUE THREAD

**Issue by AlbertoPravisano** [AlbertoPravisano/relazione-psw-ap-ab#1]
scrollando nella pagina wcag.html verso il basso la sidebar non ha altezza sufficente da coprire tutta la pagina, considerare di rimuovere il background-color: black e lasciare solo il box fluttuante grigio

**Comment by AlbertoPravisano** [comment]
risolto
